ui/app/services/path-help.js

```diff
@@ -149,10 +148,8 @@ export default Service.extend({
       //include url params
       if (params) {
         const { name, schema, description } = params[0];
-        let label = capitalize(name);
-        if (label.toLowerCase() !== 'name') {
-          label += ' name';
-        }
+        let label = capitalize(name.split('_').join(' '));
```

Comment:
Doesn't the `form-field` template already do this as a fallback? Should we only add label here if it's explicitly in the response as displayAttrs.name ?